tests/acceptance/support/sandbox.rs

```diff
@@ -44,19 +44,24 @@ impl CacheBuilder {
 
         // write expiry file
         let one_day = Duration::from_secs(24 * 60 * 60);
-        let expiry_date = HttpDate::from(if self.expired {
+        let expiry_date = Expires::from(if self.expired {
             SystemTime::now() - one_day
         } else {
             SystemTime::now() + one_day
         });
+
+        let mut header_values = Vec::with_capacity(1);
+        expiry_date.encode(&mut header_values);
+        let encoded_expiry_date = header_values.first().unwrap();
```

Comment:
```suggestion
        // Since we just `.encode()`d into `header_values, it is guaranteed to
        // have a `.first()`.
        let encoded_expiry_date = header_values.first().unwrap();
```